src/api_routes_v4.rs

```diff
@@ -222,6 +225,9 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/icon", delete().to(delete_community_icon))
           .route("/banner", post().to(upload_community_banner))
           .route("/banner", delete().to(delete_community_banner))
+          .route("/post_tag", post().to(create_community_tag))
+          .route("/post_tag", put().to(update_community_tag))
+          .route("/post_tag", delete().to(delete_community_tag))
```

Comment:
I would go with `/tags` instead, using underscore looks ugly.

Reply:
there's also underscore in some other api methods, and here i think it's good to be unambiguous

Reply:
IMO these should be at :

- POST `/community/tag` (creates)
- PUT `/community/tag` (updates)
- DELETE `/community/tag` (deletes)

Reply:
The reason I would disagree with this is that it is pretty ambiguous. These are not "community tags" they are "post tags" in a community. Community tags could just as well be tags that admins of an instance add to communities, for example a "technology" tag to all tech communities. This is something that we might want in the future, and adding ambiguity just because an "underscore looks ugly" in a place that no user will ever see seems pretty misguided to me.

Reply:
I spose the naming of these could be either where they live / who they belong to, or what they apply to. In that sense these are community tags, that apply to posts. This kind of hints that we're trying to do both at once, when we should adopt one: `.route("/**post_tag**", post().to(create_**community**_tag))`

Doesn't matter to me too much, as long as we're consistent. I think using the ownership level makes sense tho : these community tags might also apply to comments at some point, in addition to posts.

Reply:
Yes it all needs to be made more consistent. For the main tags table `community_tags` makes sense, while the relation tag-post can be represented by a table community_tags_for_post`. And the api endpoints should be `/community/tag` like mentioned above.

Reply:
I can adjust it, but I don't think what either of you are saying makes any sense. _both_ pieces of information are important. These tags _belong_ to a community but _apply_ to a post. These are not tags applied to a community.

Reply:
The tags are defined by the community so calling them community tags makes sense. And in the future it would also be possible to use the same tags for users.